projects/optic/src/commands/diff/diff.ts

```diff
@@ -241,6 +241,9 @@ const runDiff = async (
     }
 
     logComparison(specResults, { output: 'pretty', verbose: false });
+
+    console.log('');
+    console.log(`Configure check rulesets in your local optic[.dev].yml file.`);
```

Comment:
I think it might be easier for us to just name this as `optic.dev.yml` - we'll be throwing in some deprecation warnings later for optic.yml, although I dont think it matters that much -

Reply:
Yeah actually I went on changing things and adding the deprecation warning in the next pr, so I can change it now